### PR TITLE
Update on async update again

### PR DESCRIPTION
**Pull request by bvaughn**

### CODE REVIEW

File: content/blog/2018-03-27-update-on-async-rendering.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ There is a common misconception that fetching in `componentWillMount` lets you a
 >
 > In the longer term, the canonical way to fetch data in React components will likely be based on the “suspense” API [introduced at JSConf Iceland](/blog/2018/03/01/sneak-peek-beyond-react-16.html). Both simple data fetching solutions and libraries like Apollo and Relay will be able to use it under the hood. It is significantly less verbose than either of the above solutions, but will not be finalized in time for the 16.3 release.
 
+> Note
+>
```
Comment (gaearon) [inline content/blog/2018-03-27-update-on-async-rendering.md]: I think two notes look a bit odd, maybe we could combine? Although I don’t feel strongly and maybe separate is better. Just calling some extra attention to this.

File: content/blog/2018-03-27-update-on-async-rendering.md [hunk 2]
```diff
@@ -94,6 +94,10 @@ There is a common misconception that fetching in `componentWillMount` lets you a
...
+> When supporting server rendering, it's currently necessary to provide the data synchronously. `componentWillMount` was often used for this purpose, but the `constructor` can be used as a replacement. The upcoming suspense APIs will add a way to do async data fetching cleanly for both client and server rendering.
```
Comment (gaearon) [inline content/blog/2018-03-27-update-on-async-rendering.md]: One thing to note is that some people do two pass rendering to work around this. First pass kicks off the async requests, then after promises resolve they do a second pass. It’s not great but maybe worth acknowledging it somehow, or at least making sure our answer doesn’t ignore/dismiss this scenario.

**Comment by sophiebits** [comment]
Ahh sorry, I already added one already in two reckless commits:

78a2a196f76d0e96d4d36184086de531c538eb73 Update 2018-03-27-update-on-async-rendering.md 
a84d8e4e574beb9db370407554b404a3252df298 This is why you shouldn't Edit on GitHub without review

**Comment by bvaughn** [comment]
No worries, I'll revert that commit and just update the code change then.

**Pull request merged**